Source/DafnyLanguageServer.Test/Util/AssertWithDiff.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Text;
+using DiffPlex.DiffBuilder;
+using DiffPlex.DiffBuilder.Model;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Util;
+
+public class AssertWithDiff {
```

Comment:
These two projects do not have the same testing framework (one throws an XUnitException, the other one is ` Microsoft.VisualStudio.TestTools.UnitTesting`).
Alternatively, I could refactor both codes, but I'm worried that it could cause trouble to have two different testing frameworks in the same project.